examples/multigpu_advtrain/evaluator.py

```diff
@@ -0,0 +1,333 @@
+import os
+import numpy as np
+import logging
+import warnings
+import math
+
+import tensorflow as tf
+
+from cleverhans.utils_tf import _FlagsWrapper
+from cleverhans.attacks import FastGradientMethod, BasicIterativeMethod
+
+from model import MLP_probs
+from target_attack import ProjectedGradientDescentMethod
+
+
+def model_eval(sess, x, y, predictions=None, X_test=None, Y_test=None,
+               feed=None, args=None, model=None):
+    """
+    Compute the accuracy of a TF model on some data
+    :param sess: TF session to use when training the graph
+    :param x: input placeholder
+    :param y: output placeholder (for labels)
+    :param predictions: model output predictions
+    :param X_test: numpy array with training inputs
+    :param Y_test: numpy array with training outputs
+    :param feed: An optional dictionary that is appended to the feeding
+             dictionary before the session runs. Can be used to feed
+             the learning phase of a Keras model for instance.
+    :param args: dict or argparse `Namespace` object.
+                 Should contain `batch_size`
+    :param model: (deprecated) if not None, holds model output predictions
+    :return: a float with the accuracy value
+    """
+    args = _FlagsWrapper(args or {})
+
+    assert args.batch_size, "Batch size was not given in args dict"
+    if X_test is None or Y_test is None:
+        raise ValueError("X_test argument and Y_test argument "
+                         "must be supplied.")
+    if model is None and predictions is None:
+        raise ValueError("One of model argument "
+                         "or predictions argument must be supplied.")
+    if model is not None:
```

Comment:
If we do copy-paste the function, we shouldn't copy-paste the deprecated parts

Reply:
Address by 2ef5a2c.